marketplace/src/app/app.module.ts

```diff
@@ -18,6 +18,11 @@ import {BadgeModule} from "primeng/badge";
 import {MenuModule} from "primeng/menu";
 import {BrowserAnimationsModule} from "@angular/platform-browser/animations";
 import { LoginComponent } from './components/login/login.component';
+import {RippleModule} from "primeng/ripple";
+import {PasswordModule} from "primeng/password";
+import {ReactiveFormsModule} from "@angular/forms";
+import { RegisterComponent } from './components/register/register.component';
+import {DividerModule} from "primeng/divider";
 
 @NgModule({
   declarations: [
```

Comment:
You are declaring every component in the AppModule but it's better the create separte modules that bundles certain components. This way it keeps you AppModule cleaner

Reply:
Will solve this in a seperate issue